connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -94,4 +94,12 @@ private[kusto] object CslCommandsGenerator {
   def generateTableAlterRetentionPolicy(tmpTableName: String, period: String, recoverable: Boolean): String = {
     s""".alter table $tmpTableName policy retention '{ "SoftDeletePeriod": "$period", "Recoverability":"${if (recoverable) "Enabled" else "Disabled"}" }' """
   }
+
+  def generateShowTableMappingsCommand(tableName: String, kind: String): String = {
+    s""".show table $tableName ingestion $kind mappings"""
```

Comment:
tablename escaping? also, mapping using new syntax maybe? (optional: maybe move the command generator to our java client? versioning might be easier)

Reply:
i think its not really nice that every command i add ill have to add it to java first, publish the version there - and only then use it here ..